tutor-plugins/courses-visibility.py

```diff
@@ -0,0 +1,10 @@
+from tutor import hooks
+
+hooks.Filters.ENV_PATCHES.add_item(
+    (
+        "openedx-dockerfile-post-python-requirements",
+        """
+        RUN pip install --editable git+https://github.com/ghassanmas/edx-search@fix-visbile-courses#egg=edx-search
```

Comment:
I'm not sure this is an actual fix that we want to ship to the demo platform. Sure, it will certainly allow us to mark the test as solved, but it will not allow anyone else to benefit from the fix once Tutor v14 is released. If there is a bug upstream, it should be fixed in the corresponding repo.